libs/langgraph/langgraph/pregel/__init__.py

```diff
@@ -878,6 +890,12 @@ def stream(
                     },
                 }
 
+            # if we're in a subgraph and have a saved checkpoint,
```

Comment:
this doesn't seem right, means subgraphs never accept new input after their first run?

Reply:
great point, let me fix